blog/docusaurus-zielony-przyjaciel/index.mdx

```diff
@@ -0,0 +1,309 @@
+otwartego dostępu, żeby zindeksować stronę. Jeżeli to nie pasuje do naszej
+sytuacji, możemy skorzystać z innej z
+[dostępnych opcji](https://docusaurus.io/docs/search), w tym dołączyć swój
+własny mechanizm wyszukiwania.
+
+## Reuse
+
+Docusaurus pozwala na reużywać fragmenty stron za pomocą
+[importów w pliku Markdown](https://docusaurus.io/docs/markdown-features/react#importing-markdown).
+Ta opcja pozwala nam użyć ten sam fragment strony z opcjonalnymi parametrami. Na
+przykład taki fragment:
+
+```markdown title="_fruit.mdx"
+Mój ulubiony owoc to {props.fruit}!
+```
+
+Używamy go w taki sposób:
+
+```markdown
+import Fruit from './\_fruit.mdx';
+
+<Fruit fruit="🍌" />
+
+<Fruit fruit="🥦" />
+```
+
+import Fruit from './_fruit.mdx';
+
+<Fruit fruit="🍌" />
+
+<Fruit fruit="🥦" />
+
+## Tłumaczenie
+
+Docusaurus ma cały
+[wbudowany mechanizm tłumaczenia](https://docusaurus.io/docs/i18n/introduction),
+który pozwala przetłumaczyć większość stron w większości przypadków. Jedyny
+wyjątek stanowi skomplikowane użycie plików MDX, które jeszcze nie jest do końca
+proste do przetłumaczenia. Ale są sposoby, żeby to obejść.
+
+Podstawą mechanizmu tłumaczenia są dwa aspekty:
+
+- Pliki MDX są tłumaczone jak pliki Markdown. Większość agencji tłumaczeniowych
+  i narzędzi do tłumaczeń powinna sobie z tym poradzić.
+- Tłumaczenie customowych stron JSX/TSX i innego kodu możemy osiągnąć używając
+  wbudowanych komend, które ekstrahują tekst do plików JSON.
+
+## Blogowanie
+
+Jeżeli chcecie publikować nowości o Waszym projekcie, Docusaurus proponuje
+zrobić to przez moduł do blogowania. Każdy post na blogu jest plikiem MDX i
+zachowuje się bardzo podobnie do strony z docsami, ale dodatkowo ma datę.
+Posiada też tagi, które pozwalają wyświetlić listę podobnych wpisów.
+
+## JSX, React i Node.js
+
+Format MDX używany w Docusaurusie pozwala na używanie komponentów JSX lub TSX
+(Typescript), czyli elementów strony napisanych w Reakcie. Jeżeli potrafisz
+programować, lub znasz kogoś kto potrafi, to otwiera całą masę możliwości:
+
+- Konfigurowalne formularze
+- Tabele, list, lub sekcje, których treść pochodzi ze źródła danych
+- Strony generowanie w czasie budowania, na przykład zawierające listy błędów
+  lub parametrów
+- Interaktywne próbki kodu z plików, które sa testowane, żeby upewnić się, że
+  próbka działa
+- Pięknie wystylizowanie karty, galerie, karuzele i tym podobne.
+
+Docusaurus jest budowany za pomocą Node.js. To oznacza, że w momencie budowania
+mamy pełną władzę nad tym skąd pochodzi treść i możemy automatyzować. Możemy
+wszystko napisać ręcznie, ale możemy też niektóre lub wszystkie strony
+`wygenerować`. Na przykład możemy pobrać dane z bazy i wygenerować z nich
+tabele, albo zebrać wpisy na bloga z CMSa, lub release notes z Jiry. Dla
+majsterkowicza z odrobiną zapału nic nie jest niemożliwe.
+
+## Cena
+
+W momencie publikacji artykułu, Docusaurus jest "za darmo" i można go
```

Comment:
```suggestion
W momencie publikacji artykułu, Docusaurus jest darmowy i można go
```

Reply:
Słowo "za darmo" było w cudzysłowie. To było celowe? Bo taki cudzysłów sugeruje, że tak jest jakiś haczyk i nie jest naprawdę darmowy.